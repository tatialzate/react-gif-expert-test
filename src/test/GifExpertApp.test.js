import React from 'react';

const { shallow } = require("enzyme")
const { GifExpertApp } = require("../GifExpertApp")

describe('Pruebas del GifExpertApp', () => { 

    test('Debe presentarse correctamente', () => { 
        const categories = ['Dogs', 'Food'];
        const wrapper = shallow(<GifExpertApp defaultCategories={ categories } />);

        expect( wrapper ).toMatchSnapshot();
        expect( wrapper.find('GifGrid').length ).toBe( categories.length );
    })
})