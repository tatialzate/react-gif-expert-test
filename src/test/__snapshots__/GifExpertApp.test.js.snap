// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del GifExpertApp Debe presentarse correctamente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Dogs"
      key="Dogs"
    />
    <GifGrid
      category="Food"
      key="Food"
    />
  </ol>
</Fragment>
`;
